src/settings/target/manifest.rs

```diff
@@ -16,15 +15,27 @@ use serde::{Deserialize, Serialize};
 use crate::terminal::emoji;
 use crate::terminal::message;
 
+fn empty_string() -> String {
```

Comment:
i'm not convinced these should be functions- but i am open to it. otherwise looks good to me.

Reply:
serde requires them to be functions: 
```
expected function, found enum variant `some_string`
```